Tickets/sitcom 948

joaquinhernandezg opened the pull request:
Pull Request to include the latest changes in technote SITCOMTN-088
Changes:
 - Upgrade to new technote format. Old format is deprecated https://documenteer.lsst.io/technotes/migrate.html
 - Remove sections "SITCOM-797: M1M3 - Slewing analysis" "Positioning SITCOM-798: M1M3 - Settling time after a slew" and put the corresponding technote links in section "M1M3 Performance Analysis Tech Notes"
 - Add new technote https://sitcomtn-107.lsst.io/ to the listing

Introduction is still pending.

dsanmartim commented on index.rst:
```diff
@@ -1,143 +1,63 @@
-SITCOM-797: `M1M3 - Slewing analysis - Positioning <https://jira.lsstcorp.org/browse/SITCOM-797>`__ 
+   This technote summarizes the results of the M1M3 dynamic tests.
+   In addition, it serves as an index to the other tech notes produced on each test or data analysis.
+   That includes tests like: Bump Test, Hard Point Breakaway Test, Settling Time, Positioning Repeatability,
```
Bump Test and Hard Point Breakaway Test are not part of the dynamical tests if I'm not mistaken. In my conception, they are part of the quasi-static tests. Please, confirm with @b1quint if he wants to include those tests in this technote.

b1quint replied:
This is correct. However, we want to keep all the M1M3 in the same index technote. Maybe we should rename the technote to "M1M3 Tests" to be more generic.

dsanmartim commented on index.rst:
```diff
@@ -1,143 +1,63 @@
-Test Case and Results
----------------------
+* `SITCOMTN-081 <https://sitcomtn-081.lsst.io/>`_ M1M3 Hardpoint oscillations during elevation slews
+* `SITCOMTN-082 <https://sitcomtn-082.lsst.io/>`_ M1M3 Hard Point Breakaway Analysis
+* `SITCOMTN-083 <https://sitcomtn-083.lsst.io/>`_ M1M3 mirror cell bump testing
```
See comment above.

dsanmartim commented on index.rst:
```diff
@@ -1,143 +1,63 @@
-Related SITCOM tickets
-======================
+.. abstract::
+   This technote is linked with: `SITCOM-948 <https://jira.lsstcorp.org/browse/SITCOM-948>`_
```
The description of the Jira ticket should be updated according to the technotes listed in the "M1M3 Performance Analysis Tech Notes" section.

dsanmartim commented on index.rst:
```diff
@@ -1,143 +1,63 @@
-
+**LTS-88-REQ-0052**: The positioning system SHALL maintain mirror decenter less than
++/- 6 micrometer, mirror tilt less than +/- 24 e-6 degree, and piston less than +/- 1
+micrometer, all relative to the mirror cell, after a slew of 3.5 degrees or less (short slew).
```
Although Req-051 and 052 are indeed valid requirements for the dynamical tests, they are not the only ones. If you look at [sitcomtn-084](https://sitcomtn-084.lsst.io) content, you will see it also needs to comply with Req-053 and Req-054 on top of the Req-052. So, my comment is to call your attention to keeping track of the requirements listed in each technote. Maybe a good idea would be to list requirements and link them with the corresponding technote, like a summary relating a specific test (technote) with its requirement.

b1quint replied:
So, maybe instead of linking all the requirements, we should add a link to https://ls.st/LTS-88. Hopefully, Systems Engineering can help us map the requirements. However, it might be easier to highlight specific reqs in each of the more focused technotes.

dsanmartim reviewed (commented):
Hi Joaquin,

Thanks for taking care of this ticket. Here are a few comments for your consideration. BTW, nice job on migrating the technote to the new formatting!

Best,
David Sanmartim.

joaquinhernandezg commented:
Thanks @dsanmartim  and @b1quint. Acoording to your suggestions, changed the technote's title to "M1M3 Tests" and removed the detailed description of the requirements, but keeping the reference to the LTS-088 document. Is that ok?
